svgnative/CMakeLists.txt

```diff
@@ -136,12 +138,15 @@ elseif(MSVC)
     endif()
 elseif(APPLE)
     # GCC or Clang
-    set(CMAKE_CXX_STANDARD 14)
+    if (USE_SKIA)
+        set(CMAKE_CXX_STANDARD 14)
+    endif()
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -stdlib=libc++")
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -pedantic -Wextra")
 else()
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -pedantic -Wextra")
 endif()
+message(STATUS "CMAKE_CXX_STANDARD: ${CMAKE_CXX_STANDARD}")
```

Comment:
Keep? or just for debugging?

Reply:
IMO it helps to verify what the bots are doing. I'd like to keep it.